src/zjs_buffer.c

```diff
@@ -414,12 +416,12 @@ static jerry_value_t zjs_buffer(const jerry_value_t function_obj,
         jerry_size_t size = 0;
         char *str = zjs_alloc_from_jstring(argv[0], &size);
         if (!str) {
-            return zjs_error("zjs_buffer: could not allocate string");
+            return zjs_error("could not allocate string");
```

Comment:
possible leak here, shouldn't you release the new_buf and also free the *buf created above?